### PR TITLE
3 Linux distributions added: CAINE, DEFT + DEFT Zero

### PR DESCRIPTION
**Pull request by hslatman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,6 +98,9 @@ A curated list of tools and resources for security incident response, aimed to h
 ### Linux Distributions
 * [Security Onion](https://github.com/Security-Onion-Solutions/security-onion) - Security Onion is a special Linux distro aimed at network security monitoring featuring advanced analysis tools.
 * [SIFT Workstation](http://digital-forensics.sans.org/community/downloads) - The SANS Investigative Forensic Toolkit (SIFT) Workstation demonstrates that advanced incident response capabilities and deep dive digital forensic techniques to intrusions can be accomplished using cutting-edge open-source tools that are freely available and frequently updated.
+* [CAINE](http://www.caine-live.net/index.html) - The Computer Aided INvestigative Environment (CAINE) contains numerous tools that help investigators during their analysis, including forensic evidence collection.
+* [DEFT](http://www.deftlinux.net/) - The Digital Evidence & Forensics Toolkit (DEFT) is a Linux distribution made for computer forensic evidence collection. It comes bundled with the Digital Advanced Response Toolkit (DART) for Windows.
+* [DEFT Zero](http://www.deftlinux.net/) - DEFT Zero is a light edition of DEFT, focused primarily on forensically sound evidence collection.
```
Comment (meirwah) [inline README.md]: I think it's a bit of an overkill to include both DEFT and DEFT Zero...
Don't you think?

**Comment by hslatman** [comment]
Sure, they are almost the same, but Zero does not have its own website, which might make it less known for people not already familiar with DEFT itself, or the other way around, which was my main reason to include them both. Furthermore, Zero can still be considered quite a different products, as thus more agile.

**Comment by meirwah** [comment]
add a line about DEFT Zero to the DEFT entry sounds better...

**Comment by hslatman** [comment]
Merged DEFT and Zero now

**Comment by meirwah** [comment]
Looks better, but please remove second link (it points to the same link as he first one)...

File: README.md [hunk 2]
```diff
@@ -98,6 +98,8 @@ A curated list of tools and resources for security incident response, aimed to h
 ### Linux Distributions
 * [Security Onion](https://github.com/Security-Onion-Solutions/security-onion) - Security Onion is a special Linux distro aimed at network security monitoring featuring advanced analysis tools.
 * [SIFT Workstation](http://digital-forensics.sans.org/community/downloads) - The SANS Investigative Forensic Toolkit (SIFT) Workstation demonstrates that advanced incident response capabilities and deep dive digital forensic techniques to intrusions can be accomplished using cutting-edge open-source tools that are freely available and frequently updated.
+* [CAINE](http://www.caine-live.net/index.html) - The Computer Aided INvestigative Environment (CAINE) contains numerous tools that help investigators during their analysis, including forensic evidence collection.
```
Comment (meirwah) [inline README.md]: INvestigative should be Investigative

**Comment by hslatman** [comment]
INvestigative for CA*IN*E, but yeah, it may disrupt reading.

**Comment by meirwah** [comment]
:+1:

**Pull request merged**